python/tensorflowjs/converters/converter.py

```diff
@@ -303,6 +379,11 @@ def setup_arugments():
       type=bool,
       default=True,
       help='Strip debug ops (Print, Assert, CheckNumerics) from graph.')
+  parser.add_argument(
```

Comment:
It would be useful to use this flag on other conversions, but this PR is targeting using it only on layer model output, let us document and add checks.